simpledotorg/simple-server
# Update patient states to collate blood sugar data

**Story card:** [sc-7793](https://app.shortcut.com/simpledotorg/story/7793/update-the-materialized-views-to-include-diabetes-outcomes)

## Because

Collating PatientBloodSugar with ReportingPatientState is required for generating diabetes management report

## This addresses

Update the Reporting Patient States to include diabetes treatment outcomes

## Test instructions

- Checkout to the `update-patient-states-to-collate-blood-sugar-data` branch
- Run migration `bundle exec rails db:migrate`
- Create a few patients and blood sugars for these patients 
- Refresh the materialized view `bundle exec rake db:refresh_reporting_views`
- You should be able to see appropriate data in the `ReportingPatientState` table
